docs/source/configuration/authorization.mdx

```diff
@@ -644,7 +644,19 @@ This option will entirely reject queries that would have some parts filtered by
 ```yaml title="router.yaml"
 authorization:
   preview_directives:
+    enabled: true
     log_errors: true
 ```
 
-By default, when part of a query is filtered by authorization, the list of filtered paths is added to the response and logged by the router. Filtering parts of a query according to the client's rights may be seen as normal operation, and not warrant investigation by platform operators, so those logs can be disabled.
\ No newline at end of file
+By default, when part of a query is filtered by authorization, the list of filtered paths is added to the response and logged by the router. Filtering parts of a query according to the client's rights may be seen as normal operation, and not warrant investigation by platform operators, so those logs can be disabled.
```

Comment:
```suggestion
By default, when part of a query is filtered by authorization, the list of filtered paths is added to the response as top-level errors and logged by the router. Filtering parts of a query according to the client's rights may be seen as normal operation, and not warrant investigation by platform operators, so those logs can be disabled.
```

My first question when reading this was "are the filtered paths added as errors or extensions"?